Program.cs

```diff
@@ -8,33 +8,27 @@
 
 namespace myApp
 {
-    class Program
+    public class IrisData
     {
-        public class IrisData
-        {
-            [Column(0)]
-            public float SepalLength { get; set; }
-            [Column(1)]
-            public float SepalWidth { get; set; }
-            [Column(2)]
-            public float PetalLength { get; set; }
-            [Column(3)]
-            public float PetalWidth { get; set; }
-            [Column(4)]
-            public string Label { get; set; }
-        }
+        public float SepalLength { get; set; }
+        public float SepalWidth { get; set; }
+        public float PetalLength { get; set; }
+        public float PetalWidth { get; set; }
+        public string Label { get; set; }
+    }
```

Comment:
How would we make this work as desired?  Reflection isn't guaranteed to return items in source order, or at least we've tried to steer clear of such guarantees historically... are we thinking we'd back off on that?

Reply:
The only way I know of relying on property order is if you use the `StructLayout` attribute, and then sort the results by handle.  I'll find the documentation in corefx for this approach...

Reply:
See: https://github.com/dotnet/corefx/issues/14606

Reply:
Theses a shame. @jkotas can you confirm ordering of GetFields() is not guaranteed to match IL ordering?